spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -375,6 +375,15 @@ public void setExceptionHandler(ExceptionHandler exceptionHandler) {
 		this.connectionFactory.setExceptionHandler(exceptionHandler);
 	}
 
+	/**
+	 * Add custom properties to the connection factory's clientProperties.
+	 * @param properties the properties to add.
+	 * @since 1.5.6
+	 */
+	public void setCustomClientProperties(Map<String, Object> properties) {
+		this.connectionFactory.getClientProperties().putAll(properties);
+	}
```

Comment:
I wonder why existing:
````java
/**
	 * @param clientProperties the client properties.
	 * @see com.rabbitmq.client.ConnectionFactory#setClientProperties(java.util.Map)
	 */
	public void setClientProperties(Map<String, Object> clientProperties) {
		this.connectionFactory.setClientProperties(clientProperties);
	}
````
isn't enough...
Right, I see they have a different code, but shouldn't an existing method be reworked to this new logic?

Just explain, please, your intentions for these two methods.
Thaks